ssl-impl/src/main/java/io/crate/protocols/ssl/SslConfiguration.java

```diff
@@ -107,6 +107,7 @@ static SslContext buildSslContext(Settings settings) {
                     .clientAuth(ClientAuth.OPTIONAL)
                     .sessionCacheSize(0)
                     .sessionTimeout(0)
+                    .startTls(false)
```

Comment:
Not required but more explicit since we don't use StartTLS. This is not dependent on the TLS version. IMHO it doesn't hurt.